pandora-client-web/src/components/settings/permissionsSettings.tsx

```diff
@@ -76,19 +76,62 @@ function GetIcon(icon: string): string {
 	}
 }
 
-function InteractionSettings({ id }: { id: InteractionId; }): ReactElement {
+function useEffectiveAllowOthers(permissionGroup: PermissionGroup, permissionId: string): PermissionType {
+	const permissionData = usePermissionData(permissionGroup, permissionId);
+	if (permissionData?.result !== 'ok')
+		return 'no';
+
+	const {
+		permissionSetup,
+		permissionConfig,
+	} = permissionData;
+
+	if (permissionConfig != null)
+		return permissionConfig.allowOthers;
+
+	return MakePermissionConfigFromDefault(permissionSetup.defaultConfig).allowOthers;
+}
+
+function ShowEffectiveAllowOthers({ permissionGroup, permissionId }: { permissionGroup: PermissionGroup; permissionId: string; }): ReactElement {
+	const effectiveConfig = useEffectiveAllowOthers(permissionGroup, permissionId);
 	const [ref, setRef] = useState<HTMLElement | null>(null);
+
+	const { src, alt, description } = useMemo(() => {
+		switch (effectiveConfig) {
+			case 'yes':
+				return {
+					src: allow,
+					alt: 'General permission configuration preview',
+					description: 'Everyone is allowed to do this, but exceptions can be set individually.',
+				};
+			case 'no':
+				return {
+					src: forbid,
+					alt: 'General permission configuration preview',
+					description: 'No one is allowed to do this, but exceptions can be set individually.',
+				};
+			case 'prompt':
+				return {
+					src: prompt,
+					alt: 'General permission configuration preview',
+					description: 'Everyone is allowed to do this, but exceptions can be set individually.',
```

Comment:
This still has the copied description. Maybe it could be something like:

"Trying to use this permission opens a popup that lets the targeted user decide if they want to give or deny the requester this permission. Exceptions can be set individually."